ideascube/serveradmin/templates/serveradmin/name.html

```diff
@@ -0,0 +1,12 @@
+{% extends 'serveradmin/index.html' %}
+{% load i18n %}
+
+{% block twothird %}
+<h2>{% trans "Change Server Name" %}</h2>
```

Comment:
No: http://www.grammar-monster.com/lessons/capital_letters_title_case.htm

I understand why you would think that though, since typographic rules for title case in French are different (and somewhat incredible) : http://typographisme.net/post/Une-question-d%C3%A9licate-%3A-les-capitales-dans-les-titres